### ISSUE TITLE
help me about use your code to call api.openai.com via proxy server (squid) ex: http://10.1.1.1:3218

### ISSUE THREAD

**Issue by nhannguyensy** [waylaidwanderer/chatgpt-web-client#3]
help me about use your code to call api.openai.com via proxy server (squid) ex: http://10.1.1.1:3218 ,Thanks !

**Comment by waylaidwanderer** [comment]
Unfortunately, that's not supported right now unless you code your own reverse proxy server that simply applies the proxy to all requests. I'll mark this as a feature request though.

**Comment by waylaidwanderer** [comment]
Also I think you meant to open this issue on https://github.com/waylaidwanderer/node-chatgpt-api, can you confirm?

**Comment by nhannguyensy** [comment]
> Also I think you meant to open this issue on https://github.com/waylaidwanderer/node-chatgpt-api, can you confirm?

Yes, I confirm . Can you move or Do i need to create issue on that ?

**Comment by waylaidwanderer** [comment]
I'll do it.
